mesheryctl/internal/cli/root/system/update.go

```diff
@@ -44,8 +44,12 @@ var updateCmd = &cobra.Command{
 				return errors.Wrapf(err, utils.SystemError(fmt.Sprintf("failed to download %s file from %s", utils.DockerComposeFile, fileURL)))
 			}
 		}
+		err := resetMesheryConfig()
+		if err != nil {
+			return errors.Wrap(err, utils.SystemError("failed to update meshery containers"))
```

Comment:
Can this error be changed specific to the function which is called?
```suggestion
			return errors.Wrap(err, utils.SystemError("failed to reset meshery configuration"))
```

Reply:
@kushthedude I also thought the same at first, but since this function call is inside the update command if a user hits update and an error msg pops up related to resetting meshery configuration it might confuse the user. So maybe the same error message as the update function would be better. Correct me if I'm wrong.